doc/changelogs/CHANGELOG_V18.md

```diff
@@ -0,0 +1,286 @@
+
+Contributed by Michaël Zasso in [#41811](https://github.com/nodejs/node/pull/41811).
+
+#### Test Runner module (experimental)
+
+The `node:test` module facilitates the creation of JavaScript tests that report results in TAP format. To access it:
+
+`import test from 'node:test';`
+
+This module is only available under the `node:` scheme.
+
+The following is an example implementation of a parent test with two subtests:
+
+```js
+test('top level test', async (t) => {
+  await t.test('subtest 1', (t) => {
+    assert.strictEqual(1, 1);
+  });
+
+  await t.test('subtest 2', (t) => {
+    assert.strictEqual(2, 2);
+  });
+});
+```
+
+Read more in <https://nodejs.org/dist/latest-v18.x/docs/api/test.html>.
+
+Contributed by Colin Ihrig in [#42325](https://github.com/nodejs/node/pull/42325).
+
+#### Toolchain and Compiler Upgrades
+
+* Prebuilt binaries for Linux are now built on Red Hat Enterprise Linux (RHEL) 8 and are compatible with Linux distributions based on glibc 2.28 or later, for example, Debian 10, RHEL 8, Ubuntu 20.04.
+* Prebuilt binaries for macOS now require macOS 10.15 or later.
+* For AIX the minimum supported architecture has been raised from Power 7 to Power 8.
+
+Prebuilt binaries for 32-bit Windows will initially not be available due to issues building the V8 dependency in Node.js. We hope to restore 32-bit Windows binaries for Node.js 18 with a future V8 update.
+
+Node.js does not support running on operating systems that are no longer supported by their vendor. For operating systems where their vendor has planned to end support earlier than April 2025, such as Windows 8.1 (January 2023) and Windows Server 2012 R2 (October 2023), support for Node.js 18 will end at the earlier date.
+
+Full details about the supported toolchains and compilers are documented in the Node.js [BUILDING.md](https://github.com/nodejs/node/blob/v18.x/BUILDING.md#supported-platforms) file.
+
+Contributed by Richard Lau in [#42292](https://github.com/nodejs/node/pull/42292), [#42604](https://github.com/nodejs/node/pull/42604) and [#42659](https://github.com/nodejs/node/pull/42659),and Michaël Zasso in [#42105](https://github.com/nodejs/node/pull/42105) and [#42666](https://github.com/nodejs/node/pull/42666).
+
+## V8 10.1
+
+The V8 engine is updated to version 10.1, which is part of Chromium 101. Compared to the version included in Node.js 17.9.0, the following new features are included:
+
+* The [`findLast` and `findLastIndex` array methods](https://v8.dev/features/finding-in-arrays).
+* Improvements to the [`Intl.Locale` API](https://v8.dev/blog/v8-release-99#intl.locale-extensions).
+* The [`Intl.supportedValuesOf` function](https://v8.dev/blog/v8-release-99#intl-enumeration).
+* Improved performance of [class fields](https://bugs.chromium.org/p/v8/issues/detail?id=9888) and [private class methods](https://bugs.chromium.org/p/v8/issues/detail?id=10793) (the initialization of them is now as fast as ordinary property stores).
```

Comment:
```suggestion
* Improved performance of [class fields](https://bugs.chromium.org/p/v8/issues/detail?id=9888) and [private class methods](https://bugs.chromium.org/p/v8/issues/detail?id=10793) (the initialization of them is now as fast as ordinary property stores).

The data format returned by the serialization API (`v8.serialize(value)`) has changed, and cannot be deserialized by earlier versions of Node.js.
On the other hand, it is still possible to deserialize the previous format, as the API is backwards-compatible.
```

Reply:
Applied this manually in https://github.com/nodejs/node/commit/fdf002c3558577fb9b9262a1b47032d009c26167#diff-ab410adcd25ee086a4735975d4f6559c4c2b4ff831c4c9281be58e44dcba089bR119-R121 (GH wouldn't let me apply)